pyproject.toml

```diff
@@ -16,11 +16,10 @@ requests = "~=2.18"
 
 [tool.poetry.dev-dependencies]
 nose = ">=1.3.7,<2.0.0"
-rstr = "2.2.6"
 cdislogging = "^1.0.0"
```

Comment:
cdislogging is already in `dependencies`, you can remove it from `dev-dependencies`

Reply:
woops. I can do this with the publish PR.